src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue

```diff
@@ -130,9 +130,14 @@
 									{{ getVotePercentage(index) + '%' }}
 								</p>
 							</div>
-							<p v-if="selfHasVotedOption(index)" class="results__option-subtitle">
-								{{ t('spreed','You voted') }}
-							</p>
+							<div v-if="getFilteredDetails(index).length > 0 || selfHasVotedOption(index)"
+								class="results__option__details">
+								<PollVotersDetails v-if="details"
+									:details="getFilteredDetails(index)" />
+								<p v-if="selfHasVotedOption(index)" class="results__option-subtitle">
+									{{ t('spreed','You voted') }}
```

Comment:
Need to use a different string here I guess. Translation is the same as when the poll is open but you voted and e.g. the german text is unfitting to match both cases.
Maybe we need to say "You voted for this option"?